web/styles/popovers.css

```diff
@@ -89,6 +89,7 @@
 
     > .tippy-content {
         padding: 0;
+        font-size: 14px;
```

Comment:
This changes the font size for all of our popovers, right? Oh but it's overridden, yeah that's probably fine for now.